setup.cfg

```
@@ -40,6 +40,10 @@ include_package_data = True
 [options.entry_points]
 console_scripts =
     pt-os-web-portal=pt_os_web_portal.__main__:main
+    pt-os-about=pt_os_web_portal.app_window.about_app_window:main
+    pt-os-setup=pt_os_web_portal.app_window.onboarding_app_window:main
+    pt-os-tour=pt_os_web_portal.app_window.tour_app_window:main
+    pt-os-updater=pt_os_web_portal.app_window.os_updater_app_window:main
```

What if we went with
```suggestion
    open-web-portal-page=pt_os_web_portal.app_window.:main
```
and then added a new entrypoint which handles an argument for which app to open?